File: common/protcol/c#/module/center_call_servermodule.cs
/*this module file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.Collections.Generic;

namespace module
{
    public class center_call_server : juggle.Imodule 
    {
        public center_call_server()
        {
			module_name = "center_call_server";
        }

        public delegate void reg_server_sucesshandle();
        public event reg_server_sucesshandle onreg_server_sucess;
        public void reg_server_sucess(ArrayList _event)
        {
            if(onreg_server_sucess != null)
            {
                onreg_server_sucess();
            }
        }

        public delegate void close_serverhandle();
        public event close_serverhandle onclose_server;
        public void close_server(ArrayList _event)
        {
            if(onclose_server != null)
            {
                onclose_server();
            }
        }

	}
}

File: protcol/js/module/center_call_servermodule.js
/*this module file is codegen by juggle for js*/
function center_call_server_module(){
    eventobj.call(this);
    Imodule.call(this, "center_call_server");

    this.reg_server_sucess = function(){
        this.call_event("reg_server_sucess", []);
    }

    this.close_server = function(){
        this.call_event("close_server", []);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Imodule.prototype;
    center_call_server_module.prototype = new Super();
})();
center_call_server_module.prototype.constructor = center_call_server_module;

